src/include/duckdb/execution/physical_operator.hpp

```diff
@@ -51,6 +53,8 @@ class PhysicalOperator {
 
 	//! The physical operator type
 	PhysicalOperatorType type;
+	//! The memory type of the physical operator.
+	PhysicalOperatorMemoryType memory_type;
```

Comment:
Instead of storing this in the physical operator - can we store this in the `PhysicalPlan` directly (e.g. as part of the `ops` vector in a struct)?